frontend/src/Resources/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  View, Image, Text,
+  StyleSheet,
+} from 'react-native';
+import factoryIcon from '../../assets/buildings/factory.png';
+import mineIcon from '../../assets/buildings/mine.png';
+import cookieIcon from '../../assets/troop/cookie.png';
+import goldIcon from '../../assets/gold.png';
+import Colors from '../common/colors';
+import { CardView } from '../common/components';
+
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 16,
+    color: Colors.textColor,
+    fontWeight: 'bold',
+  },
+  subText: {
+    fontSize: 12,
+    color: Colors.greenColor,
+  },
+  resourceIcon: {
+    width: 14,
+    height: 14,
+    marginHorizontal: 4,
+  },
+  buildingIcon: {
+    width: 32,
+    height: 32,
+    marginHorizontal: 4,
+  },
+  rowFlex: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  resourceView: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginRight: 12,
+  },
+});
+
+function ResourceView({
+  textColor, buildingIcon,
+  resourceIcon, amount, changeRate,
+}) {
+  let changeRateStr;
+  if (changeRate < 0) {
+    changeRateStr = `${changeRate}/minute`;
```

Comment:
if the number is negative, it should be red (or orange, I'm not sure about this color, whatever you prefer)
and if it's positive, it should be green 

(so it the troops eat more food than the mine is creating, it will be for example -5/minutes, it should be orange)

![Screen Shot 2019-12-16 at 09 49 37](https://user-images.githubusercontent.com/30794349/70959919-7491b980-20b8-11ea-91bb-672cc9553bbe.png)
![Screen Shot 2019-12-16 at 09 49 42](https://user-images.githubusercontent.com/30794349/70959920-752a5000-20b8-11ea-8418-9d9453eba9dc.png)